integrationExamples/gpt/userId_example.html

```diff
@@ -2,312 +2,320 @@
 <html lang="en">
 
 <head>
-  <title>User ID Modules Example</title>
```

Comment:
@SKOCHERI can you revert the formatting changes you made to this file, so it includes only the floc-related changes?